modules/caddypki/crypto.go

```diff
@@ -39,6 +44,15 @@ func pemDecodeSingleCert(pemDER []byte) (*x509.Certificate, error) {
 	return x509.ParseCertificate(pemBlock.Bytes)
 }
 
+func pemDecodeCertificateChain(pemDER []byte) ([]*x509.Certificate, error) {
+	chain, err := pemutil.ParseCertificateBundle(pemDER)
```

Comment:
I'd rather either copy in the code of `pemutil.ParseCertificateBundle()` or remove this wrapper func entirely

Reply:
I'd still prefer this, but it's not a showstopper for now. Will await your reply in case you are OK with inlining the logic here.